docs/guides/using-organizations.md

```diff
@@ -0,0 +1,133 @@
+# Using Organizations
+
+> Note: Organizations is still under active development and requires a
+> non-standard enterprise license to use. Do not use organizations on your
+> production instance!
+>
+> For more details, [contact your account team](https://coder.com/contact).
+
+Organizations allow you to run a Coder deployment with multiple platform teams,
+all with uniquely scoped templates, provisioners, users, groups, and workspaces.
+
+## Prerequisites
+
+- Coder deployment with non-standard license with Organizations enabled
+  ([contact your account team](https://coder.com/contact))
+- User with `Owner` role
+- Coder CLI installed on local machine
+
+## Switch to the preview image and enable the experiment
+
+To try the latest organizations features, switch to a preview image in your Helm
+chart and enable the [experimental flag](../cli/server.md#--experiments).
+
+For example, with Kubernetes, set the following in your `values.yaml`:
+
+```yaml
+coderd:
+  image:
+    repo: ghcr.io/coder/coder-preview
+    tag: orgs-preview-aug-16
+  env:
+    - name: CODER_EXPERIMENTS
+      value: multi-organization
+```
+
+> See all
+> [preview images](https://github.com/coder/coder/pkgs/container/coder-preview)
+> in GitHub. Preview images prefixed with `main-` expire after a week.
+
+Then, upgrade your deployment:
+
+```sh
+helm upgrade coder coder-v2/coder -f values.yaml
+```
+
+## The default organization
+
+All Coder deployments start with one organization called `Default`.
+
+To edit the organization details, navigate to `Deployment -> Organizations` in
+the top bar:
+
+![Organizations Menu](../images/guides/using-organizations/deployment-organizations.png)
+
+From there, you can manage the name, icon, description, users, and groups:
+
+![Organization Settings](../images/guides/using-organizations/default-organization.png)
+
+## Guide: Your first organization
+
+### 1. Create the organization
+
+Within the sidebar, click `New organization` to create an organization. In this
+example, we'll create the `data-platform` org.
+
+![New Organization](../images/guides/using-organizations/new-organization.png)
+
+From there, let's deploy a provisioner and template for this organization.
+
+### 2. Deploy a provisioner
+
+[Provisioners](../admin/provisioners.md) are organization-scoped and are
```

Comment:
May add a line explaining why the new org can't use the built-in provisioner from the control plane.